feat: add examples of escaping predicates for delete (#4686)
influxdata/docs-v2: content/influxdb/v2.6/reference/syntax/delete-predicate.md

key1="value1" AND key2="value"
```

{{% note %}}
#### Predicates with special characters or keywords
If your predicate contains keywords or strings with special characters, wrap each in escaped
quotes to ensure the predicate string is parsed correctly.

```js
// Escaped due to the "-"
"_measurement=\"example-dash\""

// Escaped because "name" is a keyword
"_measurement=example and \"name\"=predicate"
```
{{% /note %}}

> {{% warn %}}

Reviewer: ```suggestion
```

key1="value1" AND key2="value"
```

> {{% warn %}}

Reviewer: ```suggestion
{{% note %}}
```

#### Predicates with special characters or keywords

> Sometimes your predicate may contain special characters or keywords. To make sure they are properly parsed, they will need to be escape quoted. See below for several examples:

Reviewer: ```suggestion
If your predicate contains keywords or strings with special characters, wrap each in escaped
quotes to ensure the predicate string is parsed correctly.
```

```
// Escaped due to the "-"
"_measurement=\"example-dash\""
```

```
// Escaped because "name" is a keyword
"_measurement=example and \"name\"=predicate"

> ```

Reviewer: ~~~suggestion
```js
// Escaped due to the "-"
"_measurement=\"example-dash\""

// Escaped because "name" is a keyword
"_measurement=example and \"name\"=predicate"
```
~~~

> 
> {{% /warn %}}

Reviewer: ```suggestion
{{% /note %}}
```

key1="value1" AND key2="value"
```

{{% note %}}
#### Predicates with special characters or keywords
If your predicate contains keywords or strings with special characters, wrap each in escaped
quotes to ensure the predicate string is parsed correctly.

```js
// Escaped due to the "-"
"_measurement=\"example-dash\""

// Escaped because "name" is a keyword
"_measurement=example and \"name\"=predicate"
```
{{% /note %}}

> 

Reviewer: ```suggestion

{{% warn %}}
```
